pkg/model/sdk_helper.go

```diff
@@ -27,6 +27,7 @@ import (
 	ackgenconfig "github.com/aws-controllers-k8s/code-generator/pkg/generate/config"
 	"github.com/aws-controllers-k8s/code-generator/pkg/names"
 	"github.com/aws-controllers-k8s/code-generator/pkg/util"
+
 	awssdkmodel "github.com/aws/aws-sdk-go/private/model/api"
```

Comment:
Tiny nit: this import should be in the second block of imports

Reply:
let me set up the goimports and create a tiny PR for this one.